src/geometry/GeoInterface.hh

```diff
@@ -49,7 +68,7 @@ struct GeoParamsPointers
 /*!
  * Data required to initialize a geometry state.
```

Comment:
Because a particle track has not just a geometry state but a physics state, particle state, etc. right? The geometry state is just a subset of a single track's state,  and the initializer there is the data required to initialize that single track's geometry state. Am I misunderstanding your confusion?

Reply:
Yes, it makes a lot more sense explained this way.  Thanks!